cmd/terramate/cli/cli.go

```diff
@@ -2404,6 +2406,7 @@ func (c *cli) gitSafeguardRemoteEnabled() bool {
 }
 
 func (c *cli) wd() string                   { return c.prj.wd }
+func (c *cli) wd2() prj.Path                { return prj.PrjAbsPath(c.rootdir(), c.wd()) }
```

Comment:
Why not more closely to what the function does?
```suggestion
func (c *cli) absPathWD() prj.Path                { return prj.PrjAbsPath(c.rootdir(), c.wd()) }
```